ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -0,0 +1,79 @@
+@using ntbs_service.Models.Enums;
+@{
+    var NotificationStatusCssClass = "";
+    switch(@Model.Notification.NotificationStatus)
+        {
+            case NotificationStatus.Draft: NotificationStatusCssClass = "notification-banner--draft";
+                break;
+            case NotificationStatus.Notified: NotificationStatusCssClass = "notification-banner--notified";
+                break;
+            case NotificationStatus.Denotified: NotificationStatusCssClass ="notification-banner--denotified";
+                break;
+            default:
+                break;
+        }
+}
+
+<dl class="notification-banner @NotificationStatusCssClass">
+    <div class="notification-banner-head">
+        <nhs-grid-row classes="notification-overview-details">
+            <nhs-grid-column grid-column-width="OneQuarter">
+                <h1 class="notification-banner-title-text">
+                    <div> @Model.Notification.NotificationStatusString </div>

```

Comment:
Do we actually need to have @Model everywhere here? I have a feeling it also works just with "Notification....", but I could be wrong. Sometimes you definitely need to explicitly mention @Model, but I haven't worked out/looked up why yet.

Reply:
Need @model here, not sure on the reason why either! If it's to do with BindProperty then this notification isn't bound so could be why.

Reply:
That's probably it!